packages/tradershub/src/modals/AddOrManageAccount/AddOrManageAccount.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment, memo, useState } from 'react';
+import ReactModal from 'react-modal';
+import { CUSTOM_STYLES } from '@/helpers';
+import { useCurrencies } from '@/hooks';
+import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { Tab, Tabs, Text } from '@deriv-com/ui';
+import CurrenciesForm from './CurrenciesForm';
+
+type TAddOrManageAccount = {
+    isOpen: boolean;
+    onClose: VoidFunction;
+};
+
+const TabTypes = {
+    0: 'CRYPTO',
+    1: 'FIAT',
+} as const;
+
+/**
+ * @name AddOrManageAccount
+ * @description The AddOrManageAccount component is used to display the Add or manage account modal.
+ * @param {TAddOrManageAccount} props - The props of the component.
+ * @param {boolean} props.isOpen - The isOpen prop is a boolean that determines if the modal is open.
```

Comment:
@shahzaib-deriv one more thing, you can delete types from JSDoc. it's not an error, but we don't need it when we use TS. I mean, you have to leave description, but we don't need `{boolean}` or `{VoidFunction}` in JSDoc, because you described it in `TAddOrManageAccount` type. You can ignore this comment, I already approved your PR :)